DOCSUP-13434: Document the serverUUID() function (#28103)
ClickHouse/ClickHouse: docs/en/sql-reference/functions/uuid-functions.md

└──────────────────┴──────────────────────────────────────┘
```

## serverUUID() {#server-uuid}

> Returns the random and unique UUID, which is generated when the server is first started and stored forever. Writes it to a file in the data directory of the server. The resultant file written is UUID and is at the same level as the status file in the ClickHouse server directory.

Reviewer: ```suggestion
Returns the random and unique UUID, which is generated when the server is first started and stored forever. The result writes to the file `uuid` created in the ClickHouse server directory `/var/lib/clickhouse/`. 
```
